<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MutiCamClass</class>
 <widget class="QMainWindow" name="MutiCamClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1050</width>
    <height>500</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1050</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1050</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MutiCam</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QListWidget" name="m_ListBoxCameras">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>261</width>
      <height>211</height>
     </rect>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::MultiSelection</enum>
    </property>
   </widget>
   <widget class="QListWidget" name="m_ListLog">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>300</y>
      <width>1041</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="m_ButtonStartStop">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>260</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Image Recording ...</string>
    </property>
   </widget>
   <widget class="QLabel" name="m_LabelStream_1">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>10</y>
      <width>385</width>
      <height>280</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="m_ColorProcessingCheckBox">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>230</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>ColorProcessing</string>
    </property>
   </widget>
   <widget class="QLabel" name="m_LabelStream_2">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>10</y>
      <width>385</width>
      <height>280</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MutiCam.qrc"/>
 </resources>
 <connections/>
</ui>
